inc/ThirdParty/Hostings/Wpengine.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Hostings;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Compatibility class for wpengine
+ *
+ * @since 3.6.1
+ */
+class Wpengine implements Subscriber_Interface {
+	/**
+	 * Array of events this subscriber wants to listen to.
+	 *
+	 * @since 3.6.1
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		if (
+			! (
+				class_exists( 'WpeCommon' )
+				&&
+				function_exists( 'wpe_param' )
+			)
+		) {
+			return [];
+		}
+
+		return [
+			'rocket_varnish_field_settings'           => 'varnish_field',
+			'rocket_display_input_varnish_auto_purge' => 'return_false',
+			'rocket_cache_mandatory_cookies'          => [ 'return_empty_array', PHP_INT_MAX ],
+			'rocket_advanced_cache_file',
+			'return_empty_string',
+			'admin_init'                              => [
+				'remove_notices',
+				'run_rocket_bot_after_wpengine',
```

Comment:
```suggestion
				[ 'remove_notices' ],
				[ 'run_rocket_bot_after_wpengine' ],
```

Reply:
You are totally right, I don't know how I didn't see that!.
Thanks I adjusted all of them